float64 dt
float64 position
float64 desired_position
float64 velocity
float64 desired_velocity
float64 commanded_effort
float64 measured_effort
